caffe2/python/tutorials/py_gen/Model_Quickload.py

```diff
@@ -10,39 +10,52 @@
 
 # # Model Quickload
 # 
-# This short notebook will show you how you can very quickly load and test SqueezeNet, which is a very small and fast model based on AlexNet and is useful for identifying objects. The range of objects groups is only 1,000.
+# This notebook will show you how you can very quickly load a pretrained SqueezeNet model and test it on images of your choice in four main steps. 
```

Comment:
Nit: This notebook will show you how you can very quickly -> This notebook will show you how to quickly